src/sentry/monitors/endpoints/organization_monitors.py

```diff
@@ -219,4 +221,41 @@ def post(self, request: Request, organization) -> Response:
         project = result["project"]
         signal_first_monitor_created(project, request.user, False)
 
+        alert_rule = request.data.get("alert_rule", {})
```

Comment:
We should probably have just a really light weight "MonitorAlertRuleValidator" type of thing that validates the simple input we accept.

This will also allow us to correctly document the fields via DRF and DRF Spectacular if the serializer is properly nested in the monitor validator.

Reply:
Let's definitely do this, we get documentation for free then.